packages/create-app/package.json

```diff
@@ -38,6 +38,7 @@
     "fs-extra": "10.1.0",
     "handlebars": "^4.7.3",
     "inquirer": "^8.2.0",
+    "node-gyp": "^9.4.0",
```

Comment:
Sorry for the mistake. Depending on the Dockerfile build flavor (i.e. external package, multistage build, backend/frontend segregation). it might be more efficient to `node-gyp` in the in the `packages/backend/package.json.hbs` dependencies.

Reply:
as per de #20113  we should move the `node-gyp` from `devDependencies` to `devdepencies`.

I noticed in that some of my clients have this package included in their custom base image, explaining why somes are note affected.

Reply:
What is the best way to proceed ?

Should i cancel this PR and make a new one ?

Reply:
No need to close this, just undo the initial change and then make the change in the correct location. Then we should be good 👍

Reply:
done